.github/workflows/js-and-android-test.yml

```diff
@@ -52,7 +52,7 @@ jobs:
                       swift:
                       - 'platform/ios/**'
             - name: Install JS packages
-              if: steps.cache.outputs.cache-hit != 'true' && (steps.changes.outputs.coreAndFeatures == 'true' || steps.changes.outputs.js == 'true')
+              if: steps.cache.outputs.cache-hit != 'true' && (steps.changes.outputs.apiAndFeatures == 'true' || steps.changes.outputs.js == 'true')
```

Comment:
These changes look like a recent PR from you - is there something wrong here?

Reply:
I was doing both PRs at the same time, essentially. I'll rebase here.